examples/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,7 @@
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="colorSecondary">@color/colorAccent</item>
```

Comment:
after migrate to 1.1.0 support library, the `setting` FAB in **example** app tints to DARK_GREEN because of the style change in 1.1.0 versus 1.0.0, so we have to define this `colorSecondary` to make it looks same as before.